mistakedetection/src/ca/mcgill/sel/mistakedetection/MistakeDetection.java

```diff
@@ -3304,10 +3320,42 @@ public static String taggerOut(String taggerInput) {
     return maxentTagger.tagString(taggerInput);
   }
 
-  private static boolean spellingMistakeCheck(String name1, String name2) {
-    int lDistance = levenshteinDistance(name1, name2);
-    return lDistance > 0 && lDistance <= MAX_LEVENSHTEIN_DISTANCE_ALLOWED;
+  private static boolean isSpelledIncorrectly(int lDistance, int studStringLength, int instStringLength) {
```

Comment:
This function should have additional documentation to make the result easier to understand.

In reality, this function has 3 logical results:

1. Word is spelled correctly: `false` is returned
2. Word has a relatively small Levenshtein distance: `true` is returned
3. Word has a large Levenshtein distance: `false` is returned

Perhaps the function could be renamed to reflect this, eg, `almostButNotExactMatch()`